src/core/drive/navigator.js

```diff
@@ -162,4 +164,10 @@ export class Navigator {
     const sameLocationRedirect = fetchResponse.redirected && fetchResponse.location.href === this.location?.href
     return sameLocationRedirect ? "replace" : "advance"
   }
+
+  #getReplaceMethodForFormSubmission(formSubmission, fetchResponse) {
+    if (this.#getActionForFormSubmission(formSubmission, fetchResponse) !== "replace") return
+    const { submitter, formElement } = formSubmission
+    return getVisitReplaceMethod(submitter, formElement) || "body"
```

Comment:
Wondering if there's a better value than `body` here. It does not have the same clear action as `morph` does. The corresponding renderer is called `PageRenderer` and the rendering is not isolated to changing the body; it also does some merging of `head` elements etc.

Reply:
It's true that the contents of `<head>` also gets merged, but the body is what gets replaced and that's the thing in question; that's what differs between the morph & non-morph case. I was keying off this description from the docs:

> During rendering, Turbo Drive replaces the current <body> element outright and merges the contents of the <head> element. The JavaScript window and document objects, and the <html> element, persist from one rendering to the next.

But I initially had this as `full`. Do you think that's better? I don't feel strongly so I'm happy to go with `full` or something else.